packages/common/src/intl/locales/ab.json

```diff
@@ -0,0 +1,20 @@
+{
+    "app.admin": "مشرف",
+    "app.customers": "عملاء",
+    "app.customer_invoices": "فواتير العميل",
+    "app.customer-invoice": "فاتورة العميل",
+    "app.customer-requisition": "طلبات الشراء",
+    "app.dashboard": "لوحة القيادة",
+    "app.logout": "تسجيل خروج",
+    "app.messages": "رسائل",
+    "app.reports": "التقارير",
+    "app.stock": "الأوراق المالية",
+    "app.suppliers": "الموردين",
+    "app.sync": "تزامن",
+    "app.tools": "أدوات",
+    "app.transactions": "المعاملات",
+    "app.welcome": "أهلا،",
+    "breadcrumb.item": "#{id}",
+    "button.ok": "نعم",
+    "button.open-the-menu": "افتح القائمة"
```

Comment:
I just copy pasted some stuff from google

Reply:
😉  It's what I would have done.
Still would like to implement a fallback in some way, so that we don't have to populate the whole list.. later though.